store.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
```

Comment:
This page looks so good! Need to sort the image and the font-family issue I flagged earlier and it'll be perfect